learners/demo_lessons.md

```diff
@@ -133,13 +133,13 @@ If you are an instructor in training and wish to use lessons from Data Carpentry
 | [Programming with R](https://swcarpentry.github.io/r-novice-inflammation/) | \* [Analyzing Multiple Data Sets](https://swcarpentry.github.io/r-novice-inflammation/03-loops-R.html) <br/> \* [Loops in R](https://swcarpentry.github.io/r-novice-inflammation/15-supp-loops-in-depth.html) <br/> \* [Making Choices](https://swcarpentry.github.io/r-novice-inflammation/04-cond.html)  | none | Later episodes depend on actions taken earlier in the lesson - consult [_Teaching Episodes with Dependencies_](#dependencies). |
-| [Using Databases and SQL](https://swcarpentry.github.io/sql-novice-survey/) | any other episode  | [Data Hygiene](https://swcarpentry.github.io/sql-novice-survey/08-hygiene/) |  |
-| [Automation and Make](https://swcarpentry.github.io/make-novice/) |  \* [Introduction](https://swcarpentry.github.io/make-novice/01-intro)<br/> \*  [Makefiles](https://swcarpentry.github.io/make-novice/02-makefiles) | \* [Conclusion](https://swcarpentry.github.io/make-novice/09-conclusion.html) | Later episodes depend on actions taken earlier in the lesson - consult [_Teaching Episodes with Dependencies_](#dependencies). |
-| [Version Control with Mercurial](https://swcarpentry.github.io/hg-novice/) |  \* [Configuring Mercurial](https://swcarpentry.github.io/hg-novice/02-configuration/)<br/> \* [Creating a Repository](https://swcarpentry.github.io/hg-novice/03-create-repo/)<br/> \* [Tracking Changes to Files](https://swcarpentry.github.io/hg-novice/04-tracking/) | \* [Automated Version Control](https://swcarpentry.github.io/hg-novice/01-basics/) | Later episodes depend on actions taken earlier in the lesson - consult [_Teaching Episodes with Dependencies_](#dependencies). |
+| [Using Databases and SQL](https://swcarpentry.github.io/sql-novice-survey/) | any other episode  | [Data Hygiene](https://swcarpentry.github.io/sql-novice-survey/08-hygiene) |  |
+| [Automation and Make](https://swcarpentry.github.io/make-novice) |  \* [Introduction](https://swcarpentry.github.io/make-novice/01-intro)<br/> \*  [Makefiles](https://swcarpentry.github.io/make-novice/02-makefiles) | \* [Conclusion](https://swcarpentry.github.io/make-novice/09-conclusion.html) | Later episodes depend on actions taken earlier in the lesson - consult [_Teaching Episodes with Dependencies_](#dependencies). |
+| [Version Control with Mercurial](https://swcarpentry.github.io/hg-novice/) |  \* [Configuring Mercurial](https://swcarpentry.github.io/hg-novice/02-configuration)<br/> \* [Creating a Repository](https://swcarpentry.github.io/hg-novice/03-create-repo)<br/> \* [Tracking Changes to Files](https://swcarpentry.github.io/hg-novice/04-tracking) | \* [Automated Version Control](https://swcarpentry.github.io/hg-novice/01-basics) | Later episodes depend on actions taken earlier in the lesson - consult [_Teaching Episodes with Dependencies_](#dependencies). |
```

Comment:
Ideally... I'd use the trailing `/`, at it's happening behind the scenes. It goes to a `index.html` page. Or to keep consistency with all, to use `/index.html`.